app/src/common/urls.js

```diff
@@ -50,4 +50,5 @@ export const URLS = {
   userPasswordRestore: () => `${urlBase}user/password/restore`,
   userSearch: () => {}, // TODO for YANA
   userSearchGlobal: () => {}, // TODO for YANA
```

Comment:
Will be nice if you complete these TODOs

Reply:
it will fix in Invite user modal PR